.gitignore

```diff
@@ -4,6 +4,7 @@ benchmarks
 cov.out
 cpu.out
 *.test
+integration_tests/BenchMarks.txt
```

Comment:
why do you add the benchmarks file in gitignore now we have `make zerobench` command to avoid updating the file when running integration tests?

Reply:
yeah i was thinking we shouldn't be committing benchmarks and should just run and check it locally.

Reply:
isnt it nice to have a benchmark file in the repo that anyone reading the repo can see?

Reply:
I remember, that during the daily we talked, that once we finish implementing the hints, we will add the benchmarks to the README, so I agree that we put it to the git ignore for now

Reply:
Oh ok got it !